### ISSUE TITLE
Adding support for placeholders in the Seed field (but perhaps other fields as well)

### ISSUE THREAD

**Issue by Eddict** [Rookiestyle/KeePassOTP#109]
situation: I already have existing One-Time Passwords in my database using the built-in functionality and just started to checkout/use this plugin to discover the added functionality.

it seems the plugin does not pick up the existing secrets (eg. custom string "TimeOtp-Secret-Base32") so i need to enter it again in the Seed field of the OTP setup form. i tried using the placeholder "{S:TimeOtp-Secret-Base32}", but that leads to an OTP error message.
wouldn't that be handy feature or am i just too novice to and something like that is already there? or perhaps it's a stupid idea for whatever reason i don't see (yet).

so basically supporting all placeholders in the [One-Time Passwords (OTPs) section](https://keepass.info/help/base/placeholders.html#plh) in the appropriate fields of this plugin.

thanks for any feedback!

**Comment by Rookiestyle** [comment]
Placeholders won't be possible.

If you want to use KeePassOTP, you can migrate the KeePass fields.
Use the KeePassOTP options for that.
More details can be found here: https://github.com/Rookiestyle/KeePassOTP/wiki/Migrate-to--and-from-other-plugins#keepass
